compysition/actors/basicauth.py

```diff
@@ -37,7 +37,7 @@ def consume(self, event, *args, **kwargs):
             try:
                 authorization = event.environment['HTTP_AUTHORIZATION']
                 user, password = base64.decodestring(authorization.split(' ')[1]).split(':')
-            except:
+            except Exception:
```

Comment:
I believe `except` and `except Exception` are the same net effect. I would recommend catching `KeyError` for the case of "No auth headers..."